src/main.js

```diff
@@ -244,3 +244,81 @@ let destinations = [
   ]
 
 
+  //visited and bucktelist lists
```

Comment:
With Parcel (and many any other modern module bundler tools) the way images are load is quite different. They need to be "required" so that the local server (localhost) know exactly where they are.

For that we use `require()` and pass the image path. Do not forget the `./` at the beginning so that Parcel knows the path is related to the `main.js`:

```diff 
  {
    city: "Rome",
    country: "Italy",
+    photo: require("./img/rome.jpg"),
    visited: true,
  },
```